Metagenomics/tutorials/mothur-miseq-sop.md

```diff
@@ -0,0 +1,1516 @@
+>**2: Merge fasta outputs**
+>
+> - **Tool:** Merge.files
+> - **Parameters:**
+>   - Set **merge** to *fasta files*
+>   - Set **inputs** to the `trim.contigs.fasta` *collection* output of the previous step
+>
+>    > ### :bulb: Collections as input
+>    > To provide a collection as input for a tool, click on the `Dataset collection` button in front
+>    > of the parameter. The drop-down menu will now list collections as possible inputs.
+>    > ![](../../shared/images/tools_collection_input.png)
+>    {: .tip}
+>
+> This will simply concatenate all the fasta files into a single file. In order to retain the knowledge
+> of which reads came from which samples, we also create a group file.
+>
+>**3: Make group file**
+>
+> - **Tool:** Make.group
+> - **Parameters:**
+>   - Set **Method** to *Automatically from collection*
+>   - Set **fasta collection** to the `trim.contigs.fasta` output of the make.contigs step
+>
+{: .hands_on}
+
+Examine the output. The *group file* consists of two columns, the first is the sequence read name, and
+the second is the group (sample) it belongs to. The sample names were generated from the dataset names
+of the pairs in our collection.
+
+```
+read_name1  sample1
+read_name2  sample1
+read_name3  sample2
+..
+```
+
+Next we want to improve the quality of our data. But first, let's get a feel of our data
+
+> ### :pencil2: Hands-on: Summarize data
+>
+> - **Tool:** Summary.seqs
+> - **Parameters:**
+>   - Set `fasta` parameter to the merged fasta file
+>   - We do not need to supply a names or count file
+>
+{: .hands_on}
+
+The `summary` output files give information per read. The `logfile` outputs also contain some summary
+statistics:
+
+```
+             Start    End        NBases     Ambigs   Polymer  NumSeqs
+Minimum:     1        248        248        0        3        1
+2.5%-tile:   1        252        252        0        3        3810
+25%-tile:    1        252        252        0        4        38091
+Median:      1        252        252        0        4        76181
+75%-tile:    1        253        253        0        5        114271
+97.5%-tile:  1        253        253        6        6        148552
+Maximum:     1        502        502        249      243      152360
+Mean:        1        252.811    252.811    0.70063  4.44854
+# of Seqs:   152360
+```
+
+This tells us that we have 152,360 sequences that for the most part vary between 248 and 253 bases.
+Interestingly, the longest read in the dataset is 502 bp. Be suspicious of this. Recall that the reads
+are supposed to be 251 bp each. This read clearly didn't assemble well (or at all). Also, note that at
+least 2.5% of our sequences had some ambiguous base calls. We'll take care of these issues in the next
+step when we run `screen.seqs`.
+
+> ### :pencil2: Hands-on: Filter reads based on quality and length
+>
+> The following tool will remove any sequences with ambiguous bases and anything longer than 275 bp.
+>
+> - **Tool:** Screen.seqs
```

Comment:
yep, all the tools are from mothur except phinch, krona, and the plotting tool at the end of tutorial